SourceX/controls/axis_direction.cpp

```diff
@@ -0,0 +1,55 @@
+#include "axis_direction.h"
+
+#include <SDL.h>
+
+namespace dvl {
+
+AxisDirection AxisDirectionRepeater::Get(AxisDirection axis_direction)

```

Comment:
Show we maybe set the default to 200 since it is only in one place that isn't the case?

or at least define them so they aren't just magical values.

Reply:
Good idea, done